presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/MetastoreOperationResult.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive.metastore;
+
+import org.joda.time.DateTime;
+
+import java.util.Collections;
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+
+public class MetastoreOperationResult
+{
+    private final List<DateTime> lastDataCommitTimes;
+
+    public static final MetastoreOperationResult EMPTY_RESULT = new MetastoreOperationResult();
+
+    public MetastoreOperationResult()
+    {
+        lastDataCommitTimes = Collections.emptyList();
```

Comment:
- Use `this(ImmutableList.of());`. `ImmutableList` should be imported from `com.google.common.collect.ImmutableList;`

Reply:
Use `this(ImmutableList.of());`